docs/generated/devkit/nrwl_devkit.md

```diff
@@ -1066,19 +1066,21 @@ Convert an Nx Generator into an Angular Devkit Schematic.
 
 ### createPackageJson
 
-▸ `Const` **createPackageJson**(`projectName`, `graph`, `options?`): `PackageJson`
+▸ `Const` **createPackageJson**(`projectName`, `graph`, `options?`, `additionalDependencies?`): `PackageJson`
 
 **`deprecated`** Import this from @nrwl/js instead
 
 #### Parameters
 
-| Name                    | Type                                                              |
-| :---------------------- | :---------------------------------------------------------------- |
-| `projectName`           | `string`                                                          |
-| `graph`                 | [`ProjectGraph`](../../devkit/documents/nrwl_devkit#projectgraph) |
-| `options`               | `Object`                                                          |
-| `options.isProduction?` | `boolean`                                                         |
-| `options.root?`         | `string`                                                          |
+| Name                      | Type                                                              |
+| :------------------------ | :---------------------------------------------------------------- |
+| `projectName`             | `string`                                                          |
+| `graph`                   | [`ProjectGraph`](../../devkit/documents/nrwl_devkit#projectgraph) |
+| `options`                 | `Object`                                                          |
+| `options.isProduction?`   | `boolean`                                                         |
+| `options.root?`           | `string`                                                          |
+| `options.target?`         | `string`                                                          |
+| `additionalDependencies?` | `string`[]                                                        |
```

Comment:
Let's call them `helperDependencies` and make them part of the options to keep the contract as similar as possible.